gulpfile.js

```diff
@@ -181,11 +182,14 @@ gulp.task("watch-build-test", ["build", "build-test"], function () {
 gulp.task("clean", function () {
     var del = require("del");
     var pathsToDelete = [
-        outPath,
-        ".vscode-test"
-    ].map(function (folder) {
-        return folder + "/**";
-    });
+        "src/**/*.js",
+        "src/**/*.js.map",
+        "test/**/*.js",
+        "test/**/*.js.map",
+        "out/",
```

Comment:
Shouldn't we just remove "out" from everywhere?

Reply:
Just a back compatibility for us.
If run `git pull` without `gulp clean` we will have a lot of unversioned files.
The same as in other places

Reply:
This will become non-actual once this brach is merged into master, but I'm fine with leaving this as-is for now